qutebrowser/browser/commands.py

```diff
@@ -303,6 +303,14 @@ def openurl(self, url=None, related=False,
                 tab = False
                 bg = True
 
+            if utils.url_in_tabs(cur_url):
```

Comment:
I don't like this method of determining if we clicked on a "buffer" in the list. However, I don't think there's an easy way to fix it without redoing a lot of things (QStandardItem seems to only take a string).

Why not place this code inside `_open` behind a config switch? Then it would apply to `quickmark-load` and `bookmark-load` for free, it wouldn't cause any overhead for people disabling it, and it would be implied this behavior would be the case globally for all opened urls.

Reply:
I moved the check to `_open` and I did get the quickmark and bookmark as expected. But, when doing a plain `:open [existing url]` it did not kick in.